content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,220 @@ We have deprecated the following cert-manager APIs:
+to store the root CA instead of the issuing CA into the `ca.crt` field for
+issued certificates. All of the information which was previously available is
+still available: the intermediate should appear as part of the chain in
+`tls.crt`.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  This is a change of behavior with the CA Issuer, make sure that the change in
+`ca.crt` does not impact you current use.
+
+{{% /pageinfo %}}
+
+# New Features
+
+## Support for the built-in CertificateSigningRequests
+
+It is [now](https://github.com/jetstack/cert-manager/pull/4064) possible to use
+the built-in Kubernetes [CertificateSigningRequest][] resources with
+cert-manager. The CA Issuer is currently the only supported issuer. The feature
+is experimental and can be enabled by adding a flag to the cert-manager
+controller. For example, with Helm:
+
+```sh
+helm install cert-manager jetstack/cert-manager \
+  --set extraArgs="{--feature-gates=ExperimentalCertificateSigningRequestControllers=true}"
+```
+
+Imagining that you have an existing CA issuer named `my-ca-issuer` in the
+default namespace. To let cert-manager sign your CSR, you (the entity creating
+the CSR) need a [specific
+role](../../usage/kube-csr/#referencing-namespaced-issuers). You can then
+create a CertificateSigningRequest:
+
+```sh
+openssl genrsa -out test.key 2048
+kubectl apply -f - <<EOF
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+metadata:
+  name: test
+spec:
+  groups:
+  - system:authenticated
+  request: $(openssl req -new -key test.key -out /dev/stdout | base64 | tr -d "\n")
+  signerName: issuers.cert-manager.io/default.my-ca-issuer
+  usages:
+  - client auth
+EOF
+```
+
+You will still need to manually approve the CSR since cert-manager does not
+approve CSRs by default
+([unlike](../../concepts/certificaterequest/#approval)
+CertificateRequests that get automatically approved by default):
+
+```sh
+kubectl certificate approve test
+```
+
+cert-manager then signs the CSR. To know more about it, see [the
+documentation](../../usage/kube-csr) on how use the built-in
+CertificateSigningRequests with cert-manager.
+
+[CertificateSigningRequest]: https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/
+
+## Vault Issuer handle of `ca.crt`
+
+The Vault issuer is now able to fill the `ca.crt` in Secrets. To do that,
+cert-manager reconstructs the certificate chain that is returned by Vault and
+uses the certificate at the start of the chain as the `ca.crt`.
+
+Note that this `ca.crt` may be an intermediate signing CA instead of the actual
+root CA.
+
+The reason why this feature did not exist previously is that finding which
+certificate is at the top of the chain is not trivial. Take a look at
+[@JoshVanL](https://github.com/JoshVanL)'s excellent
+[`ParseSingleCertificateChain`](https://github.com/jetstack/cert-manager/blob/5e2a6883c1202739902ac94b5f4884152b810925/pkg/util/pki/parse.go#L167-L244)
+to see what this is all about!
```

Comment:
I like that you're crediting Josh's code here, but it seems inappropriate for the release notes.
After all, you might argue that we should have been doing this from the start and we should be applying this cert chain parsing to the CertificateRequest CA information returned by all the issuers. 

And the fact that we're saying here that the CA may contain an intermediate just adds to the general confusion around `ca.crt` and contradicts the CA bug fix described above where we say that the `ca.crt` always contains a root crt.

Reply:
I removed this part in 827ec9e

Reply:
After [a discussion](https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1623770041428700) with Josh, I rephrased the paragraph in a1744c9. Can you take a look?